### DISCUSSION TITLE
Simulation of cavity resonant modes?

### DISCUSSION THREAD

**Discussion by SengerM** [thliebig/openEMS-Project#255] [Q&A Usage]
I have to design a cavity and would need to simulate its resonant modes. I understand this should be possible using openEMS. Am I right? What would be the suggested approach? Are there tutorials with this kind of simulation?

I have not much experience, but I would guess applying the Gaussian*sine excitation in time domain and then use Fourier transform to frequency domain should give me this?

**Comment by VolkerMuehlhaus** [comment]
This is possible, but FDTD is not very efficient for this. For a high Q cavity, you will see a lot of "ringing" in time domain after you have excited the cavity mode. 

I had done such modelling with another FDTD tool, and it works, but energy decay is veeeeeeery slow for high Q cavities.

**Reply by thliebig** [reply]
You might want to start maybe with slightly "lossy air"? This way you simulations converge quicker and you can easier fine tune your setup before committing to the full lossless simulation?

**Reply by SengerM** [reply]
Actually I want to design a cavity that has as few as possible resonances in some frequency range.
